pkg/cache/node.go

```diff
@@ -245,3 +239,12 @@ func (n *SchedulerNode) canHandle(ev events.SchedulerNodeEvent) bool {
 	defer n.lock.RUnlock()
 	return n.fsm.Can(string(ev.GetEvent()))
 }
+
+func (n *SchedulerNode) enterState(event *fsm.Event) {
+	log.Logger.Info("node state",
+		zap.String("currentState", event.Src),
+		zap.String("targetState", event.Dst),
+		zap.String("event", event.Event),
+		zap.String("nodeName", n.name),
+		zap.String("currentNodeState", n.getNodeState()))
```

Comment:
This is a double up: we have current and target state already (currentAppState).
Can we also bring this inline with the way this is done in the core? Log at a debug level and with same info order: node, source, destination, and event.